src/java/fr/paris/lutece/plugins/forms/web/entrytype/IEntryDataService.java

```diff
@@ -90,7 +90,16 @@
      *            the response to convert
      * @return the list of {@code String} objects
      */
-    Map<Integer, List<String>> responseToIterationsStrings( FormQuestionResponse formQuestionResponse );
+    List<String> responseToStrings( FormQuestionResponse formQuestionResponse );
+    
+    /**
+     * Extracts the responses from the specified form question response as a map of {@code String} objects
+     * 
+     * @param formQuestionResponse
+     *            the response to convert
+     * @return the Map of {@code String} objects
+     */
+    Map<Integer, List<String>> responseToIterationsStrings(FormQuestionResponse formQuestionResponse);
```

Comment:
you are still breaking backward compatibility by adding a new method without a default implementation. Maybe you should bring the implementation up in this interface as default ? see https://docs.oracle.com/javase/tutorial/java/IandI/defaultmethods.html

Reply:
Before working on my ticket, i create a new local branch (named like the related jira ticket) based on my develop branch ( i regularely check if my develop branch need to be updated by pulling).
When my work is done and i want to push my changes i check if there is any new changes from remote branch develop. If it's the case i switch to my develop branch, pull, then switch back to my ticket branch (in this case FORMS-828) and merge develop to it. Then i push my local branch FORMS-828 on my remote FORMS-828 branch (on my personal plugin-forms repo).
Then i create my new pull request by comparing my FORMS-828 branch with the develop remote branch.

Reply:
you should probably rebase instead of merging